code/game/objects/effects/particles_system.dm

```diff
@@ -373,3 +378,62 @@ var/list/particle_string_to_type = list(
 	icon_state = "risen"
 
 	plane = ABOVE_HUD_PLANE
+
+//ZAS DUST
+/particles/zas_dust
+	width = 96
+	height = 96
```

Comment:
setting these to 32 solves the wall noclipping but reduces the amount of visible particles - looks best with spawning=1 desu

Reply:
It solves the wall "noclipping issue" but also prevents things such as the dust trailing off onto space tiles, and causes dust specks to disappear before they can fade out. I think dust being blown up wall isn't a big issue and ends up giving it a better, more volumetric look.

![airlock](https://github.com/user-attachments/assets/ea3615cc-b219-4a8a-a472-03320b77ed90)

Reply:
holy shit that looks way better